collect_app/src/main/res/layout/storage_migration_banner.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It would be cool to pull this out as a `MaterialBanner` component but I think we can do that another time. I can look at doing thta in a similar way as we did for the `MaterialFullScreenDialog`.